crates/ere-risc0/compose.yml

```diff
@@ -0,0 +1,118 @@
+# Copied and modified from https://github.com/risc0/risc0/blob/main/compose.yml.
+
+name: ere-risc0
+services:
+  redis:
+    hostname: redis
+    image: redis:7.2.5-alpine3.19
+    restart: always
+    ports:
+      - 6379:6379
+    volumes:
+      - redis-data:/data
+
+  postgres:
+    hostname: postgres
+    image: postgres:16.3-bullseye
+    restart: always
+    environment:
+      POSTGRES_DB: taskdb
+      POSTGRES_USER: worker
+      POSTGRES_PASSWORD: password
+    expose:
+      - '5432'
+    ports:
+      - '5432:5432'
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    command: -p 5432
+
+  minio:
+    hostname: minio
+    image: minio/minio:RELEASE.2024-05-28T17-19-04Z
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    volumes:
+      - minio-data:/data
+    command: server /data --console-address ":9001"
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+      - MINIO_DEFAULT_BUCKETS=workflow
+    healthcheck:
+      test: ["CMD", "mc", "ready", "local"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  exec_agent:
+    image: ere-risc0/agent:${RISC0_VERSION}
+    runtime: nvidia
+    restart: always
+    depends_on:
+      - postgres
+      - redis
+      - minio
+    mem_limit: 4G
+    cpu_count: 4
```

Comment:
This is copied from risc0's repo, not sure if we should keep it here. I tried to remove it and it didn't affect the performance.